### PR TITLE
Refactor updateNewAccount

### PR DESCRIPTION
**Pull request by Tbaut**
- debounce is now debouncing.
- icon selection isn't lagging anymore.
- removed a direct dependency to `debounce`

What it effectively solves:

On account recovery, debounce is working, the device used to calculate the address at every key stroke. Now it's effectively debouncing (after/before):
debounce

On new account, the icon selection doesn't re-render.
I do exactly the same in each example, I click on new account and click on the 4 first icons **consecutively**. You can see the selection is jumping. (before/after)
image

### CODE REVIEW

**Comment by hanwencheng** [comment]
LGTM

File: src/screens/AccountNew.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ class AccountNewView extends React.Component {
               <AccountIconChooser
                 value={selected && selected.seed && selected.address}
                 onChange={({ address, seed, bip39 }) => {
```
Comment (hanwencheng) [inline src/screens/AccountNew.js]: Could you please also rename this function here? I forget in which PR we discussed before.

**Comment by yjkimjunior** [comment]
@Tbaut tested on iphone 6, works as expected!

**Comment by yjkimjunior** [comment]
ezgif-1-9483c7328b65

**Pull request merged**